aggregator_core/src/lib.rs

```diff
@@ -69,6 +69,7 @@ impl<H: Handler> InstrumentedHandler<H> {
     }
 
     async fn before_send(&self, conn: Conn) -> Conn {
+        let conn = self.0.before_send(conn).await;
```

Comment:
Drawing attention to this: Without this line, any before_send implementations inside of this handler would not be called. If this handler always wraps simple Fn handlers, the behavior is identical, but at some point in the future omitting any wrapped before_send calls may be confusing.